docs/index.md

```diff
@@ -0,0 +1,253 @@
+# SOCI Index
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Conceptual Data Model](#conceptual-data-model)
+  - [SOCI Index Manifest](#soci-index-manifest)
+    - [zTOC](#ztoc)
```

Comment:
nit: something got weird with this indentation and it looks like zTOC is a subsection of SOCI Index Manifest.

Reply:
Fixed in latest.